examples/imitation_learning_openbot/generate_dataset.py

```diff
@@ -35,72 +40,52 @@
     # load config
     config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
 
-    # make sure that we are not in trajectory sampling mode
+    # make sure that we are loading trajectories from a file
     config.defrost()
-    config.SIMULATION_CONTROLLER.IMITATION_LEARNING_TASK.GET_POSITIONS_FROM_TRAJECTORY_SAMPLING = False
-    config.SIMULATION_CONTROLLER.IMITATION_LEARNING_TASK.POSITIONS_FILE = os.path.abspath(args.episodes_file)
+    config.SIMULATION_CONTROLLER.IMITATION_LEARNING_TASK.LOAD_TRAJECTORY_FROM_FILE = True
+    config.SIMULATION_CONTROLLER.IMITATION_LEARNING_TASK.TRAJECTORY_LOCATIONS_FILE = os.path.abspath(args.episodes_file)
     config.freeze()
  
     # handle debug configuration (markers are only produed in Developent configuration; NOT in Shipping configuration)
+    config.defrost()
     if args.debug:
```

Comment:
- I'd rather have the default configuration live in user_config, and then only make changes if `args.debug` is true.
- I don't think RENDER_OFFSCREEN should be true in debug mode.

Reply:
Updated this.